tests/routers/test_statistics.py

```diff
@@ -0,0 +1,434 @@
+
+
+## Test base case empty database responses for each statistic endpoint.
+
+
+def test_empty_database_statistics(client):
+    response = client.get("/api/v1/statistics/target/accession/accession")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/accession/assembly")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/accession/gene")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/sequence/sequence")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/sequence/sequence-type")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/gene/category")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/gene/organism")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/gene/reference")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/gene/ensembl-identifier")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/gene/refseq-identifier")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/gene/uniprot-identifier")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/experiment/publication-identifiers")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/experiment/keywords")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/experiment/doi-identifiers")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/experiment/raw-read-identifiers")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/score-set/publication-identifiers")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/score-set/keywords")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/score-set/doi-identifiers")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/score-set/raw-read-identifiers")
```

Comment:
you could do something like iterating over a list of these strings and provide error messages along with the asserts so it's clear which one failed, i.e.  
```
for case in cases:
  response = client.get(case)
  assert response.status_code == 200, f"non-200 status code for api endpoint: {case}"
  etc.